.github/workflows/pr-ci.yml

```diff
@@ -73,13 +82,25 @@ jobs:
         registry: ghcr.io/${{ github.repository_owner }}
         username: ${{ github.event.pull_request.user.login }}
         password: ${{ secrets.GHCR_PR_TOKEN }}
-      if: github.repository_owner == 'cryostatio'
+    - name: store image amd64
```

Comment:
For this, seems like it will print twice the comment that contains the 2 images.

How about moving the `Comment test image link` step into a job that runs last? Then, build jobs can define outputs that this last "print" job can run.